draft-ietf-quic-transport.md

```diff
@@ -3140,15 +3141,15 @@ STOP_SENDING frame is received on a send stream that is already in the "Data
 Sent" state, a RST_STREAM frame MAY still be sent in order to cancel
 retransmission of previously-sent STREAM frames.
 
-STOP_SENDING can only be sent for a receive stream that has not been
+STOP_SENDING SHOULD only be sent for a receive stream that has not been
 reset. STOP_SENDING is most useful for streams in the "Recv" or "Size Known"
 states.
 
 An endpoint is expected to send another STOP_SENDING frame if a packet
-containing the frame is lost.  However, once either all stream data or a
-RST_STREAM frame has been received for the stream - that is, the stream is in
-any state other than "Recv" or "Size Known" - sending a STOP_SENDING frame is
-unnecessary.
+containing a previous STOP_SENDING is lost.  However, once either all stream
+data or a RST_STREAM frame has been received for the stream - that is, the
+stream is in any state other than "Recv" or "Size Known" - sending a
+STOP_SENDING frame is unnecessary.
```

Comment:
Above you changed it so it's SHOULD.  Should it be changed here as well?

Reply:
I think of this as justification for why it's a SHOULD above, so I'd prefer to leave this text as is.

Reply:
SHOULD doesn't make sense here.